src/api/app/views/webui/requests/change_devel_projects/new.html.haml

```diff
@@ -0,0 +1,22 @@
+- @pagetitle = 'Change Devel Project Request'
+
+.row
+  .col
+    .card.mb-3
+      = render partial: 'webui/package/tabs', locals: { project: @project, package: @package }
+      .card-body
+        .row
+          .col-12.col-md-8.col-lg-6
+            %h5 Change Devel Project Request
```

Comment:
This should be `h3` to be consistent with the other pages. We can also reuse `@pagetitle` here instead of writing the title again.

Reply:
Changed to `h3`, and `@pagetitle` reused.